### ISSUE TITLE
Change some genre 

### ISSUE THREAD

**Issue by elyaB** [xaca/coding-music#94]
We need to change some genre otherwise it will be very complicated to find what we are looking for : 
- english-jazz => jazz
- english-rock => rock
- english-rap => rap
- english-pop => pop

english-rap and spanish-rap would be in the same folders for example. This way, it would be so much simplier.

(Just imagine if we were going to add a new genre for every language !)

- I wonder for the reggaeton. We can think of an architecture of sub genres. We could do it this way :  
- reggaeton 
    |_ reggaeton-new.md
    |_ reggaeton-oldschool.md

**Comment by xaca** [comment]
@elyaB I like your architecture, the genres are a lot now, we must to organized them.

**Comment by MisoMosiSpy** [comment]
Should we have one more level for languages itself?
Folder will be names as 
/language/genre/genre.md

**Comment by elyaB** [comment]
No, it would create the same problem as before !
I'll take an exemple. The user knows what genre he wants to listen to. He doesn't know which language he wants.
If we do like this :
language/genre/genre.md
The user would have to look at *every* language. We can have like 50 different languages, it would be awful.
If we do as this : 
genre/language/language.md
The user would only have to go through his desire genre, then look at all the languages.

If you ask me, I already find it odd to classify by languages. I mean, these are the same genres. When I'm looking for music, I'm looking for a specific genre, not a language. Spotify playlists, for example, are made by genres, not languages.

**Comment by elyaB** [comment]
Like, if you take classical music, do you classify it as "Hungarian Classical" and "German Classical" ? Even though it has no lyrics ?
We don't want to go down the rabbit hole of classifying by languages.

**Comment by MisoMosiSpy** [comment]
> Like, if you take classical music, do you classify it as "Hungarian Classical" and "German Classical" ? Even though it has no lyrics ?
> We don't want to go down the rabbit hole of classifying by languages.

Sure, whatever works for majority is fine.

**Comment by xaca** [comment]
@elyaB You're right. At the beginning We only have a couple of genres, now there are a lot of list.
